### ISSUE TITLE
Feature Request | Python 2 & 3 Compatibility

### ISSUE THREAD

**Issue by jolespin** [epaull/TieDIE#3]
Are there any plans to port this to Python 3 and then make it backwards compatible?  If not, I can maybe take a crack at it.  Are there any dependencies on out-dated modules?

**Comment by epaull** [comment]
Good idea, the dependencies should be OK. Sure, that would be great if you want to take a shot at it: let me know if you run into a dependency issue. Thanks!! -Evan

**Comment by jolespin** [comment]
Off the top of your head, is there any code that assumes bytes/unicode, floor divisions, or print statements?

**Comment by epaull** [comment]
No floor divisions, that should be fine for py 2->3. Yes to print
statements, that's likely the majority of work, (and let me know if you
want me to do part of all of that, and/or the best solution to get
backwards compatibility). No encoding issues with my code but the packages
may have changed: I'm wondering about numpy's genfromtxt() function, but
likely OK as well.

-Evan
